src/Models/Hooks/useRuntimeSceneData.ts

```diff
@@ -71,6 +73,91 @@ export const useRuntimeSceneData = (
+                            visual.valueExpression,
+                            sceneVisual.twins
+                        );
+
+                        // get extensionProperties, find out if they are behaviors with transforms
+                        if (visual.valueRanges[0] !== undefined) {
+                            const valueRanges = visual.valueRanges;
+
+                            // iterate through each valueRange and determine if it applies to the current mesh
+                            // based on the currentValue of the evaluated valueExpression
+                            valueRanges.forEach((valueRange) => {
+                                const values = valueRange.values;
+                                const extensionProperties =
+                                    valueRange.visual.extensionProperties;
+                                // if the currentValue is in the determined valueRange and extensionProperties is defined,
+                                // define a CustomMeshItem as the meshId and the transform associated with the currentValue
+                                if (
+                                    extensionProperties &&
+                                    values.includes(currentValue)
+                                ) {
+                                    // only add if has transform too ... currently transform is just another value set to "true"
+                                    // (issues with "unknown" types unable to be an object)
+                                    if (extensionProperties.transform) {
+                                        //have to assume that extensionProperties will have all the right info
+                                        const transform: TransformInfo = {
+                                            rotation: {
+                                                x: extensionProperties.xRot as number,
+                                                y: extensionProperties.yRot as number,
+                                                z: extensionProperties.zRot as number
+                                            },
+                                            position: {
+                                                x: extensionProperties.xPos as number,
+                                                y: extensionProperties.yPos as number,
+                                                z: extensionProperties.zPos as number
+                                            }
+                                        };
+                                        // iterate through each mesh in the element and add its meshId to a
+                                        // TransformedElement object and assign the transform (as decided by
+                                        // extensionProperties) to that TransformedElement
+                                        // problem: who is the parent? for now, we can just do the first one ...
```

Comment:
we can add TODO: just to mark this for future reference 
```suggestion
                                        // TODO: problem: who is the parent? for now, we can just do the first one ...
```